jms/1.7/modules/ROOT/pages/index.adoc

```diff
@@ -3,285 +3,66 @@
+== JMS Messaging Support Models
 
-[[configuration_settings]]
-== Configure the Connector
+JMS supports two models for messaging:
 
-Out of the box, the JMS connector comes with a fine tuned set of default values, both for publishing and consuming messages. This means that the only requirement is for you to configure which connection should be used.
+* Point-to-point queues
+* Publish and subscribe topics
 
-This example sets a minimal connection to an in-memory ActiveMQ broker the JMS connector:
+=== Point-to-Point Queues
 
-[source,xml,linenums]
-----
-<jms:config name="JMS_Config">
-  <jms:active-mq-connection/>
-</jms:config>
-----
+In the point-to-point queuing model, a sender posts messages to a particular queue and a receiver reads those messages from that same queue. In this model, the sender knows the destination of the message and posts the message directly to it.
 
-=== Define Global Defaults
+The point-to-point queuing model operates as follows:
 
-The JMS `config` allows you to define multiple parameters that will be used as a default while consuming or publishing messages. This way, you can define a global default behavior for all the operations associated the config and then override each parameter only in the operations that require a custom behavior.
+* Only one receiver consumes the received message.
+* The sender does not need to be running at the time the receiver consumes the message, and the receiver does not need to be running at the time the message is sent.
+* Every message that is successfully processed is acknowledged by the receiver.
 
-For example, we can define a default `selector` behavior when consuming a Message in any of the operations that are associated to this config, and also a default
+=== Publish and Subscribe Topics
 
-[source,xml,linenums]
-----
-<jms:config name="JMS_Config">
-  <jms:active-mq-connection/>
-  <jms:consumer-config selector="JMSType = `INVOICE`"/>
-  <jms:producer-config timeToLive="60000"/>
-</jms:config>
-----
+The publish and subscribe model supports a sender in publishing messages to a particular message topic rather than directly to a particular queue. Receivers subscribe to consume the receive messages on a particular message topic. In this model, neither the sender nor the receiver are aware of each other. An anonymous bulletin board is a good analogy for this concept.
 
-In this case, every operation `consume` or `listener` associated to this config will use this `selector` by default, while every time we do a `publish` or `publish-consume`, the outgoing Message will have a TTL of 60 seconds.
+The publish and subscribe model operates as follows:
 
-All of this properties can be overridden at operation level when required. Let's say we need to `publish` a Message with a shorter TTL, we can do so using this config and overriding the parameter:
+* Multiple receivers consume the receive the message, including none if there are no subscribers to the topic.
```

Comment:
it sounds a little bit strange "Multiple receivers consume the receive the message"